internal/build/build.go

```diff
@@ -1,6 +1,7 @@
 package build
```

Comment:
Just wondering what the thoughts were behind putting it in `build` vs an `init` in the `main` package?

Reply:
We could do that, but since `internal/build` package is intended to modify the build configuration of the `gh` executable, and this is sort of a build configuration, I thought we could reuse an existing `init` instead of introducing a new one. The fewer `init`s in Go code, the better.

Reply:
The alternative argument I think would be "the closer the `init` is to the application entry point the easier it is to reason about". You know that `init` in the `main` package for an application is going to run before imported packages without needing to reason much about the graph.

Seems low risk to keep it here though, but including the importance of this coming before `tcell` would probably be useful to any poor souls who have to deal with this in the future.